main.py

```diff
@@ -32,6 +32,7 @@
 from club.cakebot import EmbedUtil, ServerUtil, TextCommandsUtil, Bootstrap
 from lcbools import false
 from bs4 import BeautifulSoup as HTML
+from cookies import Cookie
```

Comment:
wha

Reply:
I was thinking of maintaining [a separate module](https://github.com/cakebotpro/cookies/blob/master/Cookie/__init__.py) for the cookie feature. If you think it'd be easier we can just do everything in one place

Reply:
no its fine